assets/scss/templates/_header.scss

```diff
@@ -227,13 +225,14 @@ h5 {
   margin: $dp-spaces--lv0;
   font-weight: bold;
 }
+
 p {
   margin: $dp-spaces--lv1 $dp-spaces--lv0 $dp-spaces--lv2;
   font-size: $dp-sizing--lvl2;
 }
+
 a {
   font-size: $dp-sizing--lvl2;
-  text-transform: uppercase;
```

Comment:
This _fix_ probably breaks anything else. Did you review the history?

Reply:
If I am not wrong, you have introduced the style in 1356951f99d0b01472081e4d2f49375cb9d5b4b2 #93, could you check if this latest change does not break anything related with original commit or PR? Maybe "PRUEBALO GRATIS" button.

Reply:
BTW, _blame_ is a very useful git tool, let me know if you want to learn to use it.

Reply:
Este style solo afectaba a lo que arme del panel de notificaciones que no se implemento aun. Era algo para corregir. Antes de que lo implementemos debemos arreglar los links en el panel de notificaciones, el "PRUEBALO GRATIS" 

![uppercase](https://user-images.githubusercontent.com/46735526/57300148-93ed5080-70ac-11e9-9a5d-45fafb6fc367.jpg)

Reply:
> BTW, _blame_ is a very useful git tool, let me know if you want to learn to use it.

I would like to learn!